app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -26,8 +16,16 @@
 }
 
 .supplier-table--show-more {
+  padding-bottom: 110px;
```

Comment:
This has introduced a large gap in the desktop version of the table - if that's in the designs that's ok, I just wanted to check.

If it is in the designs, it would be good to make this a variable because it's also used in `.supplier-table--show-fewer`, and ideally we'll only want to change it in one place in the future.

Reply:
I think this is an oversight. I'll sort it

Reply:
I don't see the gap as the button and its padding don't appear on desktop, and there is already css to catch it just in case:
`.supplier-table--show-more {
  padding-bottom: 110px;

  @include cads-media-breakpoint-up(md) {
    padding-bottom: 0;
  }`

I'll come back to this one once the layout change is done, it may not be required anyway